src/bot/commands/quotes.ts

```diff
@@ -1,10 +1,14 @@
 import Discord from 'discord.js';
 import MongoService from 'src/core/services/mongo.service';
```

Comment:
Sorry missed this one before, can you change this to a relative path, to keep consistent with the rest of the codebase?